packages/uploadthing/src/_internal/logger.ts

```diff
@@ -13,7 +15,28 @@ import { UploadThingError } from "@uploadthing/shared";
 
 import { IsDevelopment } from "./config";
 
-export const withMinimalLogLevel = Config.logLevel("logLevel").pipe(
+/**
+ * Config.logLevel counter-intuitively accepts LogLevel["label"]
+ * instead of a literal, ripping it and changing to accept literal
+ */
+export const ConfigLogLevel = (
```

Comment:
this is taken from the official `Config.logLevel` and just change to accept literal instead.

see https://github.com/Effect-TS/effect/blob/a55835a5c564b19222c728c9ca8beb145066a2a5/packages/effect/src/internal/config.ts#L296-L305